tests/lib/hardware/hardware_base.py

```diff
@@ -134,11 +137,15 @@ def ansible_run_playbook(self, playbook: str,
             limit = "--limit " + ":".join([n.name for n in limit_to_nodes])
         else:
             limit = ""
+        if extra_vars:
+            extra_vars = f"--extra-vars '{extra_vars}'"
```

Comment:
I think we should evaluate the extra_vars path to either consider it as a relative path to the rookcheck dir, or an absolute path.